excercsises/2_page_object/contact.spec.js

```diff
@@ -1,11 +1,14 @@
-//var homePage = require('...');
+var contactPage = require('./pages/contact');
 
 describe('Protractor Workshop app', function() {
 
 	beforeEach(function () {
+		contactPage.get()
 	});
 
 	it('should have "Get in touch" and "Address" sections on Contact Us page', function(){
+		expect(contactPage.getInTouch().getText()).toEqual("Get in Touch");
+		expect(contactPage.getAdress().getText()).toEqual("Address");
```

Comment:
this is ok for now, exercise was about creating a page class and using it. From now on, lets try to have only one expect in the test. You can compare bigger objects if you need, arrays, jsons. Here you can have one expect like this `expect(contactPage.getSections()).toEqual(["Get in Touch", "Address"]);` and have a method that will return an array with these 'sections'. *There are different concepts, some people do multiple expect in one test, but we think one test should have one expect as it is testing one specific thing. I know that existing examples here may not suggest that.